cabal.project

```diff
@@ -17,12 +17,14 @@ repository cardano-haskell-packages
     d4a35cd3121aa00d18544bb0ac01c3e1691d618f462c46129271bccf39f7e8ee
 
 -- repeating the index-state for hackage to work around hackage.nix parsing limitation
-index-state: 2024-08-20T21:35:22Z
+index-state: 2025-01-01T23:24:19Z
 
 index-state:
-  , hackage.haskell.org 2024-08-20T21:35:22Z
-  , cardano-haskell-packages 2024-09-01T03:50:08Z
+  , hackage.haskell.org 2025-01-01T23:24:19Z
+  , cardano-haskell-packages 2025-03-01T00:00:00Z
 
+constraints:
+    plutus-core ==1.37.0.0
```

Comment:
Chances are that https://chap.intersectmbo.org/package/cardano-ledger-alonzo-1.12.0.0/ needs a revision that introduces the constraint `plutus-core < 1.38`. 🤔 But that's not something we can do in this pull request.